internal/scorecard/alpha/testdata/bundle/manifests/cache.example.com_memcacheds_crd.yaml

```diff
@@ -19,12 +19,12 @@ spec:
         apiVersion:
           description: 'APIVersion defines the versioned schema of this representation
             of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
@bharathi-tenneti how you update the mockdata?

@jmccormick2001 @fabianvf  I saw that here we have `https://github.com/operator-framework/operator-sdk/tree/6e31aaa66ea5794ca78b5c8a2066e1f65a4a0ff7/internal/scorecard/alpha/testdata/bundle` which shows be a mock data to test the scorecard feature. 

However, this mock has no been checked by the CI and can not be generated by `make generate` command. Am I right?  

Also, scorecard uses the mock data in the [test/test-framework](https://github.com/operator-framework/operator-sdk/tree/master/test/test-framework) already. So, I

- Why not use the existent mock data? Why do we need to duplicate one? 
- Note that we just need to add 1 step which would be `operator-sdk bundle create --generate-only` at the end of  [hack/generate/test-framework/gen-test-framework.sh](https://github.com/operator-framework/operator-sdk/blob/master/hack/generate/test-framework/gen-test-framework.sh) and then, the mock data will be kept updated accordingly with the changes for each or and also checked. 
- Also, to update the `test/test-framework` we just need to run `make generate`
 
PS.: IHMO: We should not duplicate the mock testdata and we need to ensure that all mock data is updated accordingly with the changes otherwise we are not using a valid data to ensure the changes in the PR. 

WDYT?

Reply:
I'd like to continue to use the scorecard v2 testdata for now, get a lot of the tests and functionality working, then come back in a separate PR to start using the mock data in test/test-framework.  This is just to allow us to figure out things before we start introducing changes that might impact the test-framework (e.g. how the scorecard metadata will live within a bundle layout for example).  We would do this before moving from alpha to the actual scorecard cmd.

Reply:
Sure @jmccormick2001. 👍  I do not think that it is the scope of PR. 
I just checked it in the review and then, thought that here would be the best place to raise the concern. 
May would be nice we track this need.

Reply:
sure thing...https://issues.redhat.com/browse/OSDK-1167

Reply:
Solved and discussed today in the Eng meeting